src/Installers/Windows/AspNetCoreModule-Setup/IIS-Setup/include.wxi

```diff
@@ -2,13 +2,7 @@
 
 <Include>
   <?define DiscoverabilityKeyRoot = "SOFTWARE\Microsoft\IIS Extensions"?>
-  <?define OOBBuildVersion = $(var.BLDVERMAJOR).$(var.BLDVERMINOR).$(var.BLDNUMMAJOR).$(var.BLDNUMMINOR)?>
```

Comment:
Note: this was causing build issues from the HostOptions stuff that are now also reusing this wxi to be able to properly get the right 64/32 bit ProgramFiles, it doesn't look like we are using OOBBuildVersion anywhere since it was fine to just remove it (as BLDVERMINOR/MAJOR aren't being defined everywhere else).  So I think this was just dead definitions